tests/unit/provisioner/test_vagrantprovisioner.py

```diff
@@ -62,8 +67,7 @@ def molecule_file(tmpdir, request):
 
     pbook = d.join(os.extsep.join(('playbook', 'yml')))
     data = [{'hosts': 'all', 'tasks': [{'command': 'echo'}]}]
-
-    pbook.write(data)
+    pbook.write(yaml.dump(data))
```

Comment:
Possibly want to use safe_dump here? Not sure how Ansible will deal with the extra info that can be output with just dump.